### PR TITLE
@brian-brazil consul sd add node meta

### PR DESCRIPTION
**Pull request by omadawn**
This is for issue #3019. Makes the consul node meta labels available as __meta_consul_node_meta_<label name>

This required updating the consul api dependency.

### CODE REVIEW

File: discovery/consul/consul.go [hunk 1]
```diff
@@ -50,6 +51,8 @@ const (
 	datacenterLabel = model.MetaLabelPrefix + "consul_dc"
 	// serviceIDLabel is the name of the label containing the service ID.
 	serviceIDLabel = model.MetaLabelPrefix + "consul_service_id"
+	// nodMetaLabelPrefix will be
```
Comment (grobie) [inline discovery/consul/consul.go]: `nodeMetaLabel`

File: vendor/vendor.json [hunk 2]
```diff
@@ -1712,5 +1724,5 @@
 			"revisionTime": "2017-03-31T20:34:26Z"
 		}
 	],
-	"rootPath": "github.com/prometheus/prometheus"
+	"rootPath": "github.com/omadawn/prometheus"
```
Comment (grobie) [inline vendor/vendor.json]: Please revert

File: discovery/consul/consul.go (hunk 1 above)
Comment (omadawn) [inline discovery/consul/consul.go]: Oops, fixed.

File: vendor/vendor.json (hunk 2 above)
Comment (omadawn) [inline vendor/vendor.json]: reverted

**Comment by grobie** [comment]
Already done in #2966.

**Pull request closed without merging**